src/components/CarTypeBar/CarTypeBar.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import CarTypeBar from "./CarTypeBar";
+
+const MockCarTypeBar = () => {
+  return (
+    <BrowserRouter>
+      <CarTypeBar />
+    </BrowserRouter>
+  );
+};
+
+describe("CarTypeBar", () => {
+  it("renders the component", () => {
+    const title = "Sedan";
+    render(<MockCarTypeBar />);
+    const spanTextSedan = screen.getByText(title, { exact: true });
+    expect(spanTextSedan).toBeInTheDocument();
+  });
+
+  it("renders the component", () => {
+    const title = "SUV";
+    render(<MockCarTypeBar />);
+    const spanTextSUV = screen.getByText(title, { exact: true });
+    expect(spanTextSUV).toBeInTheDocument();
+  });
+
+  it("renders the component", () => {
+    const title = "Hatchback";
+    render(<MockCarTypeBar />);
+    const spanTextHatchback = screen.getByText(title, { exact: true });
+    expect(spanTextHatchback).toBeInTheDocument();
+  });
```

Comment:
you can check in one test, only if all three text is being rendered, no need for seperate tests